### DISCUSSION TITLE
The official docker image tags for the current stable version are

### DISCUSSION THREAD

**Discussion by rickgitdone** [blakeblackshear/frigate#10093] [General]
since the docker images have moved into ghcr.io we can no longer search. 
In the past I have used the following tags below. I am not interested in the community supported images.. 

Are there any other current docker imaged tags posted in ghcr.io other than what is posted in the doc now available?

Used in the past from docker hub 
stable
stable-amd64

**
the current Frigate docs tags reported are
stable
stable-standard-arm64
stable-tensorrt

**Comment by NickM-27** [comment]
they are all shown in the docs https://docs.frigate.video/frigate/installation#docker

The official docker image tags for the current stable version are:

- stable - Standard Frigate build for amd64 & RPi Optimized Frigate build for arm64
- stable-standard-arm64 - Standard Frigate build for arm64
- stable-tensorrt - Frigate build specific for amd64 devices running an nvidia GPU

The community supported docker image tags for the current stable version are:

- stable-tensorrt-jp5 - Frigate build optimized for nvidia Jetson devices running Jetpack 5
- stable-tensorrt-jp4 - Frigate build optimized for nvidia Jetson devices running Jetpack 4.6
- stable-rk - Frigate build for SBCs with Rockchip SoC

**Comment by saschabrockel** [comment]
Can anybody explain to me why 0.15.0 is considered stable now and not 0.15.1? I use the `stable` tag and my Frigate runs on version `0.15.0`. What do I have to use instead of `stable` to get the latest stable version and not the beta?

**Reply by NickM-27** [reply]
see https://github.com/blakeblackshear/frigate/discussions/17702
